Makefile.common

```diff
@@ -79,6 +79,28 @@ BIN_DIR                 ?= $(shell pwd)
 DOCKER_IMAGE_TAG        ?= $(subst /,-,$(shell git rev-parse --abbrev-ref HEAD))
 DOCKER_REPO             ?= prom
 
+DOCKER_ARCHS            ?= AMD64
+
+DOCKER_AMD64_IMAGE_NAME ?= $(DOCKER_IMAGE_NAME)
+DOCKER_AMD64_BASE_IMAGE ?= quay.io/prometheus/busybox:latest
+DOCKER_AMD64_BIN_PATH   ?= .build/linux-amd64
+
+DOCKER_ARM32V7_IMAGE_NAME ?= $(DOCKER_IMAGE_NAME)-arm32v7-linux
+DOCKER_ARM32V7_BASE_IMAGE ?= quay.io/prometheus/busybox-arm32v7-linux:latest
```

Comment:
Maybe we should add a `busybox-amd64-linux` image as well, then we can get rid of this variable copy-pasta.

Reply:
@SuperQ reworked here https://github.com/prometheus/busybox/pull/23